src/streamlink/plugin/plugin.py

```diff
@@ -323,5 +323,10 @@ def get_streams(self, *args, **kwargs):
     def _get_streams(self):
         raise NotImplementedError
 
+    def stream_title(self):
```

Comment:
I think this is to deal with plugins that don't have a _get_title method. I just copied it from @chrippa's commit. Could try it without it and see if it works